mypy/messages.py

```diff
@@ -45,6 +44,7 @@
 NO_RETURN_EXPECTED = 'Return statement in function which does not return'  # type: Final
 INVALID_EXCEPTION = 'Exception must be derived from BaseException'  # type: Final
 INVALID_EXCEPTION_TYPE = 'Exception type must be derived from BaseException'  # type: Final
+RETURN_IN_GENERATOR = "'return' with value in async generator is not allowed"  # type: Final
```

Comment:
This should probably be `RETURN_IN_ASYNC_GENERATOR`, since we plausibly could have a similar message for normal generators in Python 2.